lib/si-pkg/src/spec/func.rs

```diff
@@ -77,7 +77,9 @@ pub enum FuncSpecBackendKind {
     JsValidation,
     Management,
     Map,
+    NormalizeToArray,
```

Comment:
@vbustamante do we need to regenerate the clover bindings because of this change?

Reply:
You do and don't. I wrote the import to override this so that clover doesn't have to worry about it, but we should change clover anyway